go/models/feature.go

```diff
@@ -0,0 +1,26 @@
+package models
+
+import (
+	"time"
+)
+
+type FeatureName string
+
+const (
+	FeatureNameDocument      = "document"
+	FeatureNameAuth          = "auth"
+	FeatureNameChat          = "chat"
+	FeatureNameTrip          = "trip"
+	FeatureNameNote          = "note"
+	FeatureNameTransport     = "transport"
+	FeatureNameAccommodation = "accommodation"
+	FeatureNameUser          = "user"
```

Comment:
```suggestion
	FeatureNameDocument      FeatureName = "document"
	FeatureNameAuth          FeatureName = "auth"
	FeatureNameChat          FeatureName = "chat"
	FeatureNameTrip          FeatureName = "trip"
	FeatureNameNote          FeatureName = "note"
	FeatureNameTransport     FeatureName = "transport"
	FeatureNameAccommodation FeatureName = "accommodation"
	FeatureNameUser          FeatureName = "user"
```

Sinon c'est pas considéré comme un enum